gpt_index/langchain_helpers/text_splitter.py

```diff
@@ -141,6 +156,73 @@ def split_text(self, text: str) -> List[str]:
         docs.append(self._separator.join(splits[start_idx:cur_idx]))
         return docs
 
+    def split_text_with_overlaps(self, text: str) -> List[TextSplit]:
+        """Split incoming text and return chunks."""
```

Comment:
can we just have `split_text` call this function instead of having duplicate logic?